go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/golang/mock v1.6.0
+	github.com/golang/protobuf v1.5.3
```

Comment:
@RaunakShah We should not add this back as this is deprecated.  Please remove this line.  Incorporate changes in this PR https://github.com/kubernetes-csi/external-snapshotter/pull/831/commits/c7cccaa5ac65ddc07401ff936b0bc7baadf9b080, then do a "go mod tidy".  That will not add this line back.

Reply:
Done